.pre-commit-config.yaml

```diff
@@ -13,7 +13,7 @@ repos:
       - id: fix-byte-order-marker
       - id: mixed-line-ending
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: "v0.9.10"
+    rev: "v0.11.2"
```

Comment:
This is the only non-starbase change I made, the older version wasn't aware of `[tool.ruff.lint.flake8-builtins]` addition to the pyproject.toml.